Add version flag, update dependencies (#104)
VictoriaMetrics/grafana-datasource: CHANGELOG.md

## tip

> * FEATURE: add a `--version` flag for backend datasource, update backend plugin to a latest release version. See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/68).

Reviewer: ```suggestion
* FEATURE: add `--version` flag for backend datasource binary. See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/68).
```

Author: updated
